src/components/RightSidebar/Participants/ParticipantItem.vue

```diff
@@ -19,7 +19,7 @@
 				:id="participant.actorId"
 				:key="participant.attendeeId"
 				:token="token"
-				:name="displayName"
+				:name="isEmailActor ? computedName : displayName"
```

Comment:
I guess we should always use `this.computedName` here

And drop `this.displayName` as it's not used anywhere else